### PR TITLE
Remove References to Private Beta

### PR DESCRIPTION
**Pull request by versilis**
This will remove any references to Akita's private beta in the Docker Extension. 

I've changed the external signup link to navigate to Akita's web dashboard. Additionally, I've removed links to the Docker Extension tutorial video in the help FAB.

### CODE REVIEW

File: ui/src/views/config/ConfigPage.tsx [hunk 1]
```diff
@@ -152,9 +152,7 @@ export const ConfigPage = () => {
   };
 
   const handleSignupClick = () => {
-    ddClient.host.openExternal(
-      "https://www.akitasoftware.com/beta-signup?utm_source=docker&utm_medium=link&utm_campaign=beta_from_docker"
-    );
+    ddClient.host.openExternal("https://app.akita.software/login");
```
Comment (jombooth) [inline ui/src/views/config/ConfigPage.tsx:155]: Do we want this to be for new signups? If so, we may want to use `https://app.akita.software/login?sign_up`, which will have no effect currently but will look like the below once we flip the switch 
image

**Review by jombooth — APPROVED** [review]
LGTM

File: ui/src/views/config/ConfigPage.tsx (hunk 1 above)
Comment (jombooth) [reply ui/src/views/config/ConfigPage.tsx:155]: (oh, also, I would be in favor of keeping the UTM params on the new link for tracking purposes!)

Comment (versilis) [reply ui/src/views/config/ConfigPage.tsx]: Thanks for the suggestions, this all makes sense to me. 

I've added the correct link for signups and added back the UTM params in https://github.com/akitasoftware/docker-extension/pull/34/commits/404826f9fc483986ca30e4ea0b5f5c5354dc281al

**Pull request merged**